client/landing/gutenboarding/onboarding-block/create-site/index.tsx

```diff
@@ -1,60 +1,59 @@
 /**
  * External dependencies
  */
-import React, { FunctionComponent } from 'react';
+import React, { FunctionComponent, useEffect } from 'react';
 import { useI18n } from '@automattic/react-i18n';
+import { Icon } from '@wordpress/components';
 
 /**
  * Internal dependencies
  */
-import AnimatedPlaceholder from '../animated-placeholder';
 import CreateAndRedirect from './create-and-redirect';
 import { useNewQueryParam } from '../../path';
 import './style.scss';
+/**
+ * Internal dependencies
+ */
+import { ProgressBar } from '@automattic/components';
```

Comment:
@sirreal @ockham didn't see this coming when you added this for Jetpack! :-D

Reply:
Ha. Real question: does something like this exist in `@wordpress/components`? I couldn't find it. Assumed it was okay to pull from `@automattic/`

Reply:
It's totally OK. Nothing exists in `@wordpress/components` to the best of my knowledge.